site/content/usage/21-troubleshooting.md

```diff
@@ -45,3 +45,6 @@ vpc:
       us-east-1a: {id: subnet-33333333}
       us-east-1b: {id: subnet-44444444}
 ```
+
+### Deletion issues
+If your delete does not work, or you forget to add -wait on the delete, you may need to go to use amazon's other tools to delete the cloudformation stacks. This can be accomplished via the gui or with the aws cli.
```

Comment:
nit: surround `-wait` with  ` ` `

Reply:
"you may need to go to use amazon's other tools" -> "you may need to use amazon's other tools"

Reply:
should it be `--wait` ? :)